tapes/packages.tape

```diff
@@ -1,9 +1,15 @@
 package sockeye :: .versioner=git .repo="/home/hltcoe/mpost/code/sockeye" .ref=HEAD { }
 package sacrebleu :: .versioner=pip .package="sacrebleu" .tag="1.2.12" { }
 package subword_nmt :: .versioner=pip .package="subword-nmt" .tag="0.3.5" { }
-package sentencepiece :: .versioner=pip .package="sacrebleu" .tag="1.2.12" { }
 package mosesdecoder :: .versioner=git .repo="https://github.com/moses-smt/mosesdecoder" .ref=HEAD { }
 
+package sentencepiece :: .versioner=git .repo="https://github.com/google/sentencepiece" .ref="refs/tags/v0.1.6" {
+  mkdir build
```

Comment:
I would rather keep the python `sentencepiece` here as different people could have very different C++ environment settings (for example my setup on the grid requires that I always add prefix). If people have C++ version installed and prefer to use the C++ version, they could modify this to use the C++ version instead.

Reply:
Oh bummer. I forgot that python version dose not have cmd-line tools. Let's keep this.

Reply:
I realized that prefix is not the showstopper for this line, as you didn't call make install.

Did you test this? My compilation runs but it throws segmentation fault when I call `spm_train`. Other binaries seems to be working though.

Reply:
Falling back to v0.1.5 works for me. I'm going to merge with v0.1.5. Please keep it unless you have major objections.